DEVELOP.rst

```diff
@@ -23,6 +23,15 @@ Run the unit tests like so::
 
 .. _Gradle: https://gradle.org/
 
+Vulnerability Check
+===================
+
+You should run this command to check if any dependency has a vulnerability::
+
+    $ ./gradlew dependencyCheckAnalyze
+
+So you should run it whenever you're doing any dependency changes in build.gradle
```

Comment:
I think another good usage of this is running it periodically, vulnerabilities are mostly found after some version was released and used afaik.
Maybe we should run this task always as a pre-compile task instead similar on how we do style checks?

Reply:
Ah hm I think this task may run quite long (it downloads the NVD database) so running it as a hook may not be acceptable. Maybe we should just leave this advice/sentence  out?

Reply:
Removed this advice (and yes you're right, it should be run on-demand)